demisto_sdk/commands/format/format_module.py

```diff
@@ -88,14 +94,23 @@ def format_manager(input: str = None,
         verbose (bool): Whether to print verbose logs or not
         update_docker (flag): Whether to update the docker image.
         assume_yes (bool): Whether to assume "yes" as answer to all prompts and run non-interactively
+        deprecate (bool): Whether to deprecate the entity
+        use_git (bool): Use git to automatically recognize which files changed and run format on them
+        prev_ver (str): Against which branch should the difference be recognized
+        include_untracked (bool): Whether to include untracked files when checking against git
     Returns:
         int 0 in case of success 1 otherwise
     """
+    prev_ver = prev_ver if prev_ver else 'demisto/master'
+    supported_file_types = ['json', 'yml', 'py', 'md']
+    use_git = use_git or not input
+
     if input:
-        files = get_files_in_dir(input, ['json', 'yml', 'py', 'md'])
-    else:
-        files = [file['name'] for file in
-                 get_changed_files(filter_results=lambda _file: not _file.pop('status') == 'D')]
```

Comment:
is `get_changed_files` still needed? used in another place?
also, i dont really understand the filter that was used here, can you explain how is it covered in `git_util.get_all_changed_files` ?